boards/arm/thingy52_nrf52832/board.c

```diff
@@ -77,3 +79,5 @@ DEVICE_DEFINE(ccs_vdd_pwr_ctrl_init, "", pwr_ctrl_init, NULL, NULL,
 	      CONFIG_BOARD_CCS_VDD_PWR_CTRL_INIT_PRIORITY, NULL);
 
 #endif
+
+#endif
```

Comment:
That's not what I meant.

We don't want an empty .c file either.

In CMakeLists.txt you can do:

if(CONFIG_GPIO)
zephyr_library()
zephyr_library_sources(board.c)
endif()

Reply:
Ok, I did not know about this functionality in CmakeLists.